package.xml

```diff
@@ -2,67 +2,22 @@
 <package format="2">
   <name>twist</name>
   <version>0.0.0</version>
-  <description>The twist package</description>
+  <description>The TWIST package</description>
 
-  <!-- One maintainer tag required, multiple allowed, one person per tag -->
-  <!-- Example:  -->
-  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="david@todo.todo">david</maintainer>
+  <maintainer email="emanuel@todo.todo">emanuel</maintainer>
+  <maintainer email="luisfiliperdias@gmail.com">luis</maintainer>
   <maintainer email="mikael@todo.todo">mikael</maintainer>
 
+  <license>GPLv3</license>
 
-  <!-- One license tag required, multiple allowed, one license per tag -->
-  <!-- Commonly used license strings: -->
-  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
-  <license>TODO</license>
+  <author email="mikael@todo.todo">Mikael?</author>
```

Comment:
Can you provide data for author @nmssilva? :p

Reply:
well, authors and maintainers in this case are the same, we can remove the authors tag, to simplify.

Reply:
done